examples/migrations/graphiti-to-okf/scripts/run_all.sh

```
@@ -0,0 +1,105 @@
+AGENT_ID="${MEMANTO_AGENT_ID:-graphiti-okf-demo}"
+mkdir -p data data/validation okf_bundle_sample
+
+step() { echo; echo "════════════════════════════════════════"; echo "▶ $*"; echo "════════════════════════════════════════"; }
+
+# ---- backend ---------------------------------------------------------------
+if [[ "${GRAPHITI_BACKEND:-neo4j}" == "neo4j" ]]; then
+  if command -v docker >/dev/null 2>&1; then
+    step "Starting Neo4j (docker compose)"
+    docker compose up -d
+    echo "Waiting for Neo4j bolt..."
+    for i in $(seq 1 60); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)